stacker/actions/build.py

```diff
@@ -0,0 +1,201 @@
+import logging
+
+from . import base
```

Comment:
nit: Since we only use one thing out of base, might as well:

`from .base import BaseAction`